source/apiVolontaria/apiVolontaria/views.py

```diff
@@ -250,8 +250,8 @@ def post(self, request):
             )
         # We have multiple token with the same key (impossible)
         else:
-            error = 'The system have a problem, please contact us, ' \
-                    'it is not your fault.'
+            error = _('The system have a problem, please contact us, ' \
+                    'it is not your fault.')
```

Comment:
Since we're already modifying it, we'll correct:
- Double quotes instead of single
- have -> has
- No backslash
```
"The system has a problem, please contact us,"
"it is not your fault."
```